packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -64,6 +64,28 @@ function parseNumber(
   }
 }
 
+/**
+ * Parses a LogLevelString as a LogLevel
+ * @param key
+ * @param environment
+ * @param values
+ */
+function parseLogLevel(
+  key: keyof ENVIRONMENT,
+  environment: ENVIRONMENT_MAP | ENVIRONMENT,
+  values: ENVIRONMENT_MAP
+) {
+  const value = values[key];
+  const logLevelStrings = Object.keys(LogLevel).filter(
+    k => isNaN(Number(k)) === true
```

Comment:
What is this filter doing? looking for keys of the enum that aren't numbers?

Reply:
right, I almost put a comment, guess I should have `Object.keys(LogLevel)` returns not only `INFO` and such but also `0` `1` `2` `3`